package.json

```diff
@@ -31,7 +32,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
-    "@stitches/react": "canary",
+    "@stitches/react": "0.0.3-canary.4",
```

Comment:
I had to fix this to latest alpha because they've been publishing new betas and as I've busted the yarn lock we were getting the new API. I don't want to do this work now which will muddy the waters of what this PR is about.